src/lm_buddy/jobs/ragas/ragas_config.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+from pathlib import Path
+
+from pydantic import Field, field_validator
+from ragas.metrics import (
+    answer_relevancy,
+    context_precision,
+    context_recall,
+    faithfulness,
+)
+from ragas.metrics.base import MetricWithLLM
+
+from lm_buddy.integrations.huggingface import AutoModelConfig
+from lm_buddy.integrations.wandb import WandbRunConfig
+from lm_buddy.types import BaseLMBuddyConfig
+
+
+class RagasConfig(BaseLMBuddyConfig):
+    """misc settings for ragas eval job"""
+
+    metrics: list[MetricWithLLM] = [
+        faithfulness,
+        answer_relevancy,
+        context_recall,
+        context_precision,
+    ]
+
+    # path to store the generated ratings/evaluations of each dataset sample
+    result_path: str | Path | None = None
+
+
+class RagasvLLMJudgeConfig(BaseLMBuddyConfig):
```

Comment:
is this specific to vLLM, or can it take any hosted server? if it can, we can make this more general than vLLMJudge

Reply:
Ragas specifies a bunch of endpoints that are openai api compatible, makes sense to change it to openai api endpoint since that's more precise